chess-AI/experimental-work/output_representation.py

```diff
@@ -239,5 +249,5 @@ def main():
     # is being parsed correctly both for uci -> indices and indices -> uci
     # Functions should be inverse of each other.
 
-if __name__ == "__main__":
-    main()
+if __name__ == "__main__": 
```

Comment:
No whitespace at end of line: "[Do use whitespace after a comma, semicolon, or colon, except at the end of the line.](https://google.github.io/styleguide/pyguide.html#36-whitespace)"